VIDEO_3_SCRIPT.md

```diff
@@ -0,0 +1,60 @@
+
+Another important step the initiative took towards its consolidation was to partner with Postman at the end of 2020, a leading collaborative platform developing APIs and a reference in the sector worldwide. This partnership allows it to drive the growth of more and better tools to help engineers create and maintain asynchronous APIs while using their usual languages and working environments.
+
+AsyncAPI currently aims to accomplish three main goals:
+
+1. Unify all API specifications. No one does event-driven architecture alone. Most developers complement the event architecture with REST (OpenAPI), GraphQL, and RPC APIs.
+
+2. Improve the performance of the specification to make it more and more seamless. Users should be able to create their first API in a matter of minutes without prior knowledge of AsyncAPI.
+
+3. Reach the first position among the most used API specifications. This will require a growth in the number of community members of around 400% over the next two years.
+
+## How did it come about?
+
+AsyncAPI started as a side project at Hitch (API Changelog) and was later founded by Fran Mendez. The AsyncAPI developers identified a lack of tools in the message-driven space. Something was missing in the documentation specifications when presented with an inversely great toolset for HTTP APIs.
+
+They began to search for a solution for standardized message-driven specification systems from this identified need. 
+
+They formulated an "API theory,” so to speak, essentially, with as many interactive API types as RESTful web services, GraphQL, and message-based architectures, AsyncAPI would serve as a common unifying language for the various formats, protocols, and specifications. All of the above results in standardized communication in the message-based system.
+
+## What AsyncAPI brings
+
+It's an extensible, protocol-agnostic communication protocol specification that helps us in any message-based case, primarily hosted in the Internet of Things (IoT), among other use cases. AsyncAPI's main contribution to the world of APIs is the standardization of communication between APIs, making messages (events) more machine-readable. This allows developers who wish to implement it to be minimized and simplified. 
+
+Having defined, documented, and accepted the communication between APIs provides us with much higher and more efficient information control. It is a multichannel tool, so it currently supports existing protocols.
+
+Therefore, the conglomerate of AsyncAPI tools allows accelerating the development of applications. Its mechanism for automating complex and essential tasks facilitates code generation, documentation generation, validators, etc. It could be said that this is a specification that facilitates, simplifies, and accelerates the work of developers working with APIs. 
+
+## How it Works / Working case study
+Case use
```

Comment:
I actually want to remove this section because I think that **use case(s)** should be it's own video. 🤔 

This video is supposed to be about `What is AsyncAPI`, and getting into use cases and how it has been applied in other places kinda leads into it's own separate topic. Hence, I think this needs to be it's own video.

Also, I am wondering who would be good to ask to explain a good use case to us so as to make a "case study" video. How about folks from our TSC that work at other companies and join our SIG meetings often?